.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @phndiaye @tgallice @Hegazy360 @JulienVannier66 @upfluence/frontend
```

Comment:
Hmmm, it means that all of you folks are going to be asked for review for a PR ? You're sure about that ? :)

Reply:
@jlevesy i added the people from the frontend team. wasn't that the purpose ?

Reply:
Well here, each time a PR is opened, the 4 of you are going to be systematically named as a reviewers.

In the proposal, we proposed 2 codeowners per repository  + the team that will name 1 external reviewer, loadbalanced. 

For example :)

```suggestion
* @phndiaye @Hegazy360 @upfluence/frontend
```